packages/python/plotly/plotly/express/_core.py

```diff
@@ -1412,7 +1412,14 @@ def build_dataframe(args, constructor):
                 wide_cross_name = "__x__" if wide_y else "__y__"
 
     if wide_mode:
-        value_name = _escape_col_name(columns, "value", [])
+        if "labels" in args:
+            if "y" in args["labels"]:
+                y_value = args["labels"]["y"]
+            else:
+                y_value = "value"
```

Comment:
```suggestion
        if "labels" in args and "y" in args["labels"]:
            y_value = args["labels"]["y"]
```
Just a little simplification :)